release-policies.md

```diff
@@ -0,0 +1,57 @@
+# Release Policies
+
+The .NET team uses the following policies for [.NET releases](releases.md). They define what you can expect when you use a given .NET release.
+
+## Release cadence
+
+New major .NET versions are released annually in November. .NET 5.0 was the first version to be released according to this schedule. Minor releases may also be released, with no pre-defined or regular schedule.
+
+## Release types
+
+Each .NET release is defined (prior to initial release) as either **Long Term Support (LTS)** or **Current**. The difference is support time frame, as defined below:
+
+* **LTS** releases are supported for three years. They are intended for users that want the stability and lower cost of maintaining an application on a single (major.minor) .NET version for an extended period.
+* **Current** releases are supported for (typically) fifteen months. They are intended for users that want to take advantage of the newest features and improvements and to stay on the leading edge of .NET innovation. Current release users need to upgrade to later .NET releases more often to stay in support.
+
+LTS and Current releases have many similarities. The .NET team follows the same software engineering and release processes for both release types, including for security, compatibility, and reliability. Both releases may contain major new features and breaking changes. The .NET team aspires to enable straightforward migration from one release to another (LTS or Current, in either direction), and has processes in place to achieve that intention.
+
+## Servicing periods
+
+.NET Core releases are supported -- during the servicing period -- according to the following policies. Servicing policies are the same for LTS and Current releases.
+
+A single "bug bar" is used to decide if a change is warranted and safe for servicing updates. A given fix is often applied to multiple [servicing branches](https://github.com/dotnet/core/blob/master/daily-builds.md#servicing-releases), independent of release type. Breaking changes are not accepted during servicing, except (in the very rare case) to resolve a security vulnerability.
+
+Improvements are released as "patches". Patch releases are cumulative. Patches are often released on the Microsoft "Patch Tuesday" (second Tuesday of each month), but not exclusively so, and there is no guarantee that there will be a .NET release on any given Patch Tuesday. Patches are announced on the [.NET blog](https://devblogs.microsoft.com/dotnet/). A digest of monthly releases is published to [dotnet/announcements](https://github.com/dotnet/announcements/labels/Monthly-Update).
```

Comment:
"not exclusively so" is no long true - we *have* to ship on patch Tuesday on account of MU (they will not let us ship on an arbitrary day).

Reply:
Ah, good info. I didn't know. Fixed.